baselines/disable-packing-test/package.json

```diff
@@ -24,7 +24,7 @@
     "test": "c8 mocha build/test"
   },
   "dependencies": {
-    "google-gax": "^2.2.0"
+    "google-gax": "next"
```

Comment:
here too

Reply:
yes, we're waiting for this to become alive